lib/src/base-styles/_nx-tables.scss

```diff
@@ -10,141 +10,180 @@
+    > .nx-cell {
+      &:first-child, &:last-child {
+        border-radius: 0;
+      }
     }
+  }
 
-    .nx-cell--label {
-      @include semi-bold();
+  // create a stacking context so the headers don't appear over other content outside the table
+  position: relative;
+  z-index: 0;
 
-      color: $nx-text-color;
-    }
+  .nx-cell--header {
+    position: sticky;
+    top: 0;
+    z-index: 1;
   }
 }
 
-.nx-table-row--header {
-  border-bottom: $nx-border-row-dark;
+.nx-table--fixed-layout {
+  table-layout: fixed;
+
+  .nx-cell--icon {
+    box-sizing: content-box;
+
+    // the width:1px that works well with auto-layout tables is taken literally in fixed-layout tables.
+    // Instead, we make a reasonable guess at the width of the icon here.
+    width: 1em;
+    height: auto;
+
+    &:last-child {
+      // in case the icon is less than 1em wide, still put it the correct distance from the right edge
+      // (typical case: chevrons)
+      text-align: right;
+    }
+  }
 }
 
 .nx-cell {
   @include container-vertical;
   @include container-horizontal;
 
-  background-clip: padding-box;
-  border-top: $nx-border-row;
-  box-sizing: border-box;
-  color: $nx-text-color;
-  display: table-cell;
-  height: 39px; // 38px cell, 1px top border
-  padding: $nx-spacing-xs $nx-spacing-sm;
-  position: relative;
-  text-align: left;
+  padding: $nx-spacing-sm $nx-spacing-md;
+  vertical-align: top;
+}
 
-  &.nx-error {
-    border-color: $nx-fail-color-bg;
-    border-bottom-color: $nx-fail-color;
-    color: $nx-text-color-error;
-    text-align: center;
-  }
+.nx-cell--label {
+  @include semi-bold();
 }
 
 .nx-cell--header {
-  @include semi-bold;
-  border-bottom: $nx-border-row-dark;
-  background-color: $nx-off-white;
+  @include bold;
+  background-color: #bbc1de;
   border-top: none;
+  box-sizing: border-box;
 
-  color: $nx-text-color;
-  font-size: 12px;
-  height: 24px;
-  line-height: 12px;
+  color: $nx-text-color-dark;
+  font-size: $nx-font-size-s;
+  height: 48px;
   text-align: left;
   text-transform: uppercase;
-}
-
-.nx-cell--label {
-  @include semi-bold();
-
-  color: $nx-text-color;
+  white-space: nowrap;
+  vertical-align: middle;
 }
 
 // columns whose content is numerical
 .nx-cell--num {
   text-align: center;
 }
 
-// Used when table contents are empty
-.nx-cell--empty {
-  @include regular();
 
-  color: $nx-text-color;
+.nx-cell--icon {
+  color: #2c2c2c;
   text-align: center;
-}
 
-// special cells that usually contain a button or indicator - usually on the right side
-.nx-cell--chevron {
-  color: $nx-text-color;
-  padding: 0 $nx-spacing-sm;
-  text-align: right;
-  width: 14px;
-  max-width: 32px;
-  min-width: 14px;
-}
-
-.nx-cell--icon {
-  color: $nx-text-color;
-  padding: 0 $nx-spacing-sm;
-  text-align: right;
-  width: 14px;
-  max-width: 32px;
-  min-width: 14px;
+  // seems there's still no cleaner way to get a cell to shrink to fit its contents
+  width: 1px;
```

Comment:
If you add multiple icons they appear vertically. This doesn't seem optimal. Are you thinking that in that case each icon would have a separate cell?
<img width="251" alt="Screen Shot 2020-09-29 at 4 38 27 PM" src="https://user-images.githubusercontent.com/1991443/94613255-55909c80-0272-11eb-878c-dea03cbb851c.png">

Reply:
Or better yet could we have it adapt to multiple icons?

Reply:
> Or better yet could we have it adapt to multiple icons?

I've separated out the styles for the chevron column vs general icon columns, allowing general icon columns to now contain multiple icons. 9cd195e0